weave-js/src/components/Panel2/ChildPanelExportReport/ChildPanelExportReport.tsx

```diff
@@ -1,15 +1,38 @@
-import React from 'react';
+import React, {useState} from 'react';
 import _ from 'lodash';
 
 import {Alert} from '../../Alert.styles';
 import {Button} from '../../Button';
 import {Tailwind} from '../../Tailwind';
 import {useCloseDrawer, useSelectedPath} from '../PanelInteractContext';
+import {ReportSelection} from './ReportSelection';
+import {ChildPanelFullConfig} from '../ChildPanel';
+import {EntityOption, ReportOption} from './utils';
 
-export const ChildPanelExportReport = () => {
+type ChildPanelExportReportProps = {
+  config: ChildPanelFullConfig;
```

Comment:
**suggestion (non-blocking):** maybe we call this `rootConfig`? Later in this comp we'll also need to `useNodeValue` on it to get the "full" config, then `getConfigByPath` for the "target" config (AKA "local" config, as we've been calling it)... basically just trying to think ahead and make these configs easier to distinguish from each other 😅

Reply:
Changed - question is the type still `ChildPanelFullConfig` when you were testing out the local vs full config?

Reply:
Yes `ChildPanelFullConfig`! Sorry another confusing naming situation here 😅  `ChildPanelFullConfig` means it has this schema specifically:

```
{
  vars: Frame;
  input_node: NodeOrVoidNode;
  id: string;
  config: any;
}
```

... as opposed to `ChildPanelConfig`, which could be undefined to just a node/void node.

what I meant by "root" vs "full" config in my prev comment is "root" config is the very top level entry-point config that just references an artifact, while "full" config is what you get from fetching the artifact and contains _actual_ details about all the panels inside. maybe I need to come up with a different word than "full 😅

Reply:
@connieelee ahhh okay i think i am finally getting it. The root is kind of a like a meta node?

Reply:
yes yes exactly